packages/react-vapor/src/components/button/examples/ButtonExamples.1.use.md

```diff
@@ -0,0 +1,37 @@
+-   An **icon with a label does not need alternative text**. In this case, the icon is a visual help to identify the purpose faster and filling it will cause a screen reader to repeat the text twice.
+
+## Variations
+
+The Button Type choice is defined by the function of the button (e.g. primary or a secondary action), the number of actions available in the screen (e.g. only one, two or more actions) and its location (e.g. displayed in a header, action bar, page or footer).
+
+| Type      | Purpose                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Primary   | Primary Buttons are used to enable the most important actions, that is those that lead to the completion of the main task (e.g., Save, Continue, Submit), or strong call to actions. Ideally, there should be only one Primary Button per screen to avoid dilluting the user's attention.                                                                                                                                                                                                  |
+| Secondary | Secondary Buttons are used for main actions that users might want to execute but that are not in direct relation to the main task that is to be done in the screen or interface.                                                                                                                                                                                                                                                                                                           |
+| Combo     | The Combo Button got an attached drop menu because a choice of action has to be made (2 or more). They are usually used when one single action allows to create, delete or modify many different elements, which can be chosen in the dropdown list that appears on click. While using a combo button allows to avoid adding an extra step in the process, it can add friction to the user's decision making process if the options in the dropdown list are not intuitive enough to them. |
```

Comment:
Suggestion:
**Combo Buttons** have an attached drop menu where a choice between multiple actions has to be made (2 or more). Those buttons are useful when one single action allows users to create, delete or modify many different elements, which can be selected in the dropdown list that appears upon clicking. While using a **Combo Button** prevents adding an extra step in the process, it can add friction to the user decision making process when the options in the dropdown list are not intuitive enough.